force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -176,7 +207,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+        DOuble sum = (a + b + c);
+        Double average = sum / 3;
+        return average; // Replace null with the variable you used to store the result
```

Comment:
For practice, try to return this in one line, noticing that the method already returns a double